app/styles/app.scss

```diff
@@ -1,6 +1,24 @@
 // @import 'yoga-sass/assets/yoga';
 // @import 'normalize-css/normalize';
 
-// $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
+
+@import 'bulma/bulma';
+
+$fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
 @import 'font-awesome/scss/font-awesome';
 @import 'box-sizing';
+//
+body {
```

Comment:
Should not need to add any of these styles to the body tag. With bulma's `section` and `container` noted above, you can get things spot on.